purefa/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,22 @@
+name: PureFA
+files:
+- name: purefa.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/openmetrics
+  - template: instances
+    options:
+    - name: use_openmetrics
+      description: |
+        Use the latest OpenMetrics implementation for more features and better performance.
+        If running on Agent v6, you must set the `python_version` option in `datadog.yaml` to `"3"`.
+      value:
+        example: true
+        type: boolean
+      enabled: true
+    - template: instances/openmetrics
+      overrides:
+        openmetrics_endpoint.value.example: http://localhost:9491/metrics/flasharray/array?endpoint=array01
+        openmetrics_endpoint.display_priority: 1
+        collect_histogram_buckets.hidden: true
```

Comment:
Just curious why is this hidden?

Reply:
No idea - can set it to false

Reply:
Can you remove that entirely? It is false by default.

Reply:
Done